fv3core/stencils/map_single.py

```diff
@@ -93,257 +90,175 @@ def lagrangian_contributions(
         q2_adds += q2_tmp
 
 
-def compute(
-    q1: FloatField,
-    pe1: FloatField,
-    pe2: FloatField,
-    qs: FloatField,
-    mode: int,
-    i1: int,
-    i2: int,
-    j1: int,
-    j2: int,
-    kord: int,
-    qmin: float = 0.0,
-    version: str = "stencil",
-):
-    remap_profile_k = utils.cached_stencil_class(RemapProfile)(
-        kord, mode, cache_key=f"map_profile_{kord}_{mode}"
-    )
-    dp1, q4_1, q4_2, q4_3, q4_4, origin, domain, i_extent, j_extent = setup_data(
-        q1, pe1, i1, i2, j1, j2
-    )
-    q4_1, q4_2, q4_3, q4_4 = remap_profile_k(
-        qs, q4_1, q4_2, q4_3, q4_4, dp1, i1, i2, j1, j2, qmin
-    )
-    do_lagrangian_contributions(
-        q1,
-        pe1,
-        pe2,
-        q4_1,
-        q4_2,
-        q4_3,
-        q4_4,
-        dp1,
-        i1,
-        i2,
-        j1,
-        j2,
-        kord,
-        origin,
-        domain,
-        version,
-    )
-    return q1
+class LagrangianContributions:
+    """
+    Fortran name is lagrangian_contributions
```

Comment:
There isn't really a Fortran equivalent to this class, it's just the mechanics of remap_z, map1_ppm, map_scalar, and mapn_tracer, reordered to fit in a gt4py stencil

Reply:
Changed, thank you!